package.json

```diff
@@ -2,9 +2,25 @@
   "name": "cross-domain-safe-weakmap",
   "version": "1.0.10",
   "description": "Cross domain safe WeakMap.",
-  "main": "index.js",
+  "main": "dist/cross-domain-safe-weakmap.js",
+  "module": "dist/module",
   "scripts": {
-    "test": "gulp test"
+    "build": "npm run babel & npm run webpack && npm run flow:build",
```

Comment:
Also I don't think a test before building is a good idea...  What if you just want to know if it builds correctly without going through a million tests?